doc/manuals/user/walkthrough_apiv2.md

```diff
@@ -724,6 +724,12 @@ Let's examine in detail the different elements included in the payload:
     way only to show the enormous flexibility of subscriptions.
 -   You can leave `conditions.attrs` empty to make a notification
     trigger on any entity attribute change (regardless of the name of the attribute).
+-   You can make Orion mark the attributes in the notification that at the same
+    time were part of the triggering update. In addition, Orion can mark the attributes
+    that actually changed as a consequence of that update. These marks are based on
+    metadata, described in [the following piece of documentation](metadata.md). Have
+    a look at the "Notification metadata marks" section in the
```

Comment:
This has been changed. Now the reference is to the metadata section (another section in the Orion manual).

Fixed in 9a645cd8244a7c4023dc0eaaee610c87c3f351df